spot_rl_experiments/spot_rl/utils/skill_executor.py

```diff
@@ -420,13 +420,23 @@ def execute_skills(self):  # noqa: C901
         elif skill_name == "place":
             rospy.set_param("skill_in_execution_lock", True)
             print(f"current skill_name {skill_name} skill_input {skill_input}")
+            skill_input_list = skill_input.split(";")
+            receptacle = skill_input
+            if len(skill_input_list) == 3:
+                receptacle = skill_input_list[
+                    2
+                ]  # "skill_input=1_pink donut plush toy, on, 117_sofa"
+            else:
+                pass
```

Comment:
What is the alternate here? If `skill_input` is not split based on `;` what does it have?

Reply:
Skill input will always be split over `;`
We had added this else for React since for my system REACT only used to send receptacle info instead of all 3.

This is `else` block is actually redundant and has not been tested. Removing this.